docs/types/value-types.rst

```diff
@@ -59,15 +59,15 @@ This means that, for example ``~int256(0) == int256(-1)``.
 Shifts
 ^^^^^^
 
-The result of a shift operation has the type of the left operand. The
-expression ``x << y`` is equivalent to ``x * 2**y``, and, for positive integers,
-``x >> y`` is equivalent to ``x / 2**y``. For negative ``x``, ``x >> y``
-is equivalent to dividing by a power of ``2`` while rounding down (towards negative infinity).
-Shifting by a negative amount throws a runtime exception.
+The result of a shift operation has the type of the left operand, truncating the result to match the type.
+
+- For positive and negative ``x`` values, ``x << y`` is equivalent to ``x * 2**y``.
+- For positive and negative ``x`` values,  ``x >> y`` is equivalent to ``x / 2**y``, with values rounded down towards negative infinity.
```

Comment:
What about this?
```suggestion
- For positive ``x`` values,  ``x >> y`` is equivalent to ``x / 2**y``. For negative ``x`` values, ``x >> y`` is equivalent to ``(x + 1) / 2**y - 1`` (which is the same as dividing ``x`` by ``2**y`` while rounding down towards negative infinity)
```
(See https://github.com/ethereum/solidity/blob/0f7b1b31a180af9bc224971c57d2b34098ffbb23/libsolidity/ast/Types.cpp#L1201 for the argument that this is in fact true)

Reply:
@ekpyron I need to look at this inline and think about it, so will accept suggestion, but might change it slightly :)